Description: use lzo2 instead of lzo1 for LzoCompressor
Forwarded: no
Author: Thomas Girard <thomas.g.girard@free.fr>
Last-Update: 2012-12-31

--- a/TAO/tao/Compression/lzo/LzoCompressor.mpc
+++ b/TAO/tao/Compression/lzo/LzoCompressor.mpc
@@ -1,7 +1,7 @@
 //$Id: LzoCompressor.mpc 91840 2010-09-17 20:23:03Z mitza $
 
-project(LzoCompressor) : taolib, tao_output, install, compression, taoidldefaults, lzo1 {
-  requires += lzo1
+project(LzoCompressor) : taolib, tao_output, install, compression, taoidldefaults, lzo2 {
+  requires += lzo2
   sharedname   = TAO_LzoCompressor
   dynamicflags += TAO_LZOCOMPRESSOR_BUILD_DLL
 
